server/pbench/bin/index-pbench

```diff
@@ -245,15 +270,18 @@ def convert_to_float(d):
     else:
         pass
 
+class ResultData(PbenchData):
 
-class ResultData(object):
     def __init__(self, ptb, experiment, opctx):
```

Comment:
Can you use `super` here?  Something like:

```
def __init__(self, ptb, experiment, opctx):
    start, end = ptb.gen_run_timestamps()
    super(ResultData, self).__init__(start, end)
    self.run_metadata = _dict_const([
            ("runtstamp", ptb.start_run),
            ("runid", ptb.md5sum),
            ("experiment", experiment)
        ])
    self.ptb = ptb
    self.counters = Counter()
    opctx.append(_dict_const(object="ResultData", counters=self.counters))
    self.json_files = None
```

Reply:
Done.